fake_switches/juniper/juniper_netconf_datastore.py

```diff
@@ -133,8 +133,8 @@ def interface_to_etree(self, port):
         if port.description is not None:
             interface_data.append({"description": port.description})
 
-        if port.shutdown is not None:
-            interface_data.append({("disable" if port.shutdown else "enable"): ""})
+        if port.shutdown is not None and port.shutdown:
```

Comment:
What I meant is that you don't need to check for None since python will evaluate this to False
so if shutdown should suffice.